assets/app.js

```diff
@@ -737,6 +770,16 @@ new Vue({
         }
     },
     methods: {
+        validateRejectedProduct: function (type, product) {
+            if(product['rejected_'+type]){
+                if(product[type] == null || Object.keys(product[type]).length === 0 || product.length === 0){
+                    return true
+                }else{
+                    return false
+                }
+            }
```

Comment:
what does it return in else case? when `if(product['rejected_' + type])` evaluates to false.